### DISCUSSION TITLE
Where to find API docs?

### DISCUSSION THREAD

**Discussion by jmkozko** [TypedProject/ts-express-decorators#1729] [Q&A] [answered]
Links for annotations in the getting started guide are unavailable. Clicking on a decorator such as "Property" under:
https://tsed.io/docs/model.html
routes the reader to a [api-doc.tsed.io](api-doc.tsed.io) site, which is unreachable:
https://api-doc.tsed.io/api/specs/schema/types/decorators/common/Property.html

Attempting to look up the Model & JsonSchema page under:
https://github.com/tsedio/tsed/blob/production/docs/getting-started/README.md?plain=1#L76
also routes to a "page not found":
https://github.com/tsedio/tsed/blob/production/docs/model.md

Where can I go to find working API docs?

**Comment by Romakita** [comment, marked as answer]
Hellos @jmkozko 
The doc deploiment has failed yesterday. I’ll try to fix that this morning.

see you
